### PR TITLE
🍱 Add Icons and Credits File

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
- add icons in .svg format
- add credits file for icons and library
- build own RESTful API from scratch

### CODE REVIEW

File: public/mayonnaise.svg [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (HorstFrank) [inline public/mayonnaise.svg:57]:
```suggestion
```

**Review by HorstFrank — COMMENTED** [review]
GJ!

**Review by mrana17 — APPROVED** [review]
Lgtm 🚀🚀

**Review by DNaunin — APPROVED** [review]
Great Job! 🍩

**Pull request merged**